Indicators/IndicatorExtensions.cs

```diff
@@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
             return second;
         }
 
+        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
+            where T :BaseData
+            where TSecond : IndicatorBase<IndicatorDataPoint>
+        {
+            SimpleMovingAverage _smaADX = new SimpleMovingAverage("SMA of " + left.Name, period).Of(left);
+            return _smaADX;
```

Comment:
Thanks, that's what I thought, but I wanted to follow your style.  :-)

On Sat, Apr 16, 2016 at 3:10 PM, Jared <notifications@github.com> wrote:

> In Indicators/IndicatorExtensions.cs
> <https://github.com/QuantConnect/Lean/pull/379#discussion_r59974881>:
>
> > @@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
> >              return second;
> >          }
> >
> > +        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
> > +            where T :BaseData
> > +            where TSecond : IndicatorBase<IndicatorDataPoint>
> > +        {
> > +            SimpleMovingAverage _smaADX = new SimpleMovingAverage("SMA of " + left.Name, period).Of(left);
> > +            return _smaADX;
>
> Typically _ is reserved for class privates, "smaADX" is fine here. The
> underscore in the issue description was my mistake :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/QuantConnect/Lean/pull/379/files/ccf1d4a664c7b27d07ff81210a939900b2542f0b#r59974881>
>



-- 

Best regards,

Mike Ellertson
Email: mdellertson@gmail.com